cli/app.go

```diff
@@ -361,6 +377,15 @@ var app = &cli.App{
 			Aliases: []string{"q"},
 			Usage:   "suppress warnings",
 		},
+		&cli.StringFlag{
+			Name:  profileFlag,
+			Usage: "Specify a particular profile for the current command",
```

Comment:
```suggestion
			Usage: "specify a particular profile for the current command",
```
[nit] I _think_ the other usage texts are lower case?